crates/diesel_models/src/call_back_mapper.rs

```diff
@@ -0,0 +1,28 @@
+use diesel::{Identifiable, Insertable, Queryable, Selectable};
+use serde::{self, Deserialize, Serialize};
+use serde_json;
+
+use crate::schema::call_back_mapper;
+
+#[derive(
+    Clone, Debug, Eq, PartialEq, Identifiable, Queryable, Selectable, Serialize, Deserialize,
+)]
+#[diesel(table_name = call_back_mapper,  primary_key(id), check_for_backend(diesel::pg::Pg))]
+
+pub struct CallBackMapper {
+    pub id: String,
+    #[serde(rename = "type")]
+    pub type_: String,
+    pub data: serde_json::Value,
```

Comment:
```suggestion
    pub data: pii::SecretSerdeValue,
```

Reply:
if there is no sensitive value, is this still needed? Also this table should be used for storing HS entity reference